### DISCUSSION TITLE
Plus symbol in file paths

### DISCUSSION THREAD

**Discussion by kendalldita** [dita-ot/dita-ot#4341] [Q&A]
I haven't been able to reproduce this, but I've seen + symbol being interpreted as space in args.input.dir. Do you think it makes sense to change the input URI to use %2B where + occurs, if the input is a file and the corresponding character is a space?

**Comment by jelovirt** [comment]
> change the input URI to use %2B where + occurs, if the input is a file and the corresponding character is a space

So if the input is `foo+bar` and the corresponding file name is `foo bar`, the input should be changed to `foo%2Bfoo`?

**Reply by kendalldita** [reply]
Possibly. I didn't find where it was happening and it then stopped. There were messages saying something like:

Processing file:/C:/tmp/this that/abc123.dita

When arg.input was a map and args.input.dir is C:/tmp/input/this+that which contains abc123.dita

These both create C:\tmp\this+that\abc123.dita:

    new File(new URI("file:/C:/tmp/this%2Bthat/ab123.dita")).createNewFile();
    new File(new URI("file:/C:/tmp/this+that/ab123.dita")).createNewFile();

The + as space comes from html form encoding, I think. So, it shouldn't be space character.
